specification/compute/resource-manager/Microsoft.Compute/ComputeRP/stable/2023-07-01/examples/virtualMachineScaleSetExamples/VirtualMachineScaleSetVM_AttachDetachDataDisks_MaximumSet_Gen.json

```diff
@@ -0,0 +1,52 @@
+{
+  "parameters": {
+    "resourceGroupName": "rgcompute",
+    "vmScaleSetName": "aaaaaaaaaaaaaa",
+    "instanceId": "aaaaaaaaaaaa",
+    "api-version": "2023-07-01",
+    "subscriptionId": "{subscription-id}",
+    "parameters": {
+      "attachDataDisks": [
+        {
+          "lun": 1,
+          "id": "/subscriptions/{subscription-id}/resourceGroups/myResourceGroup/providers/Microsoft.Compute/disks/vmss3176_vmss3176_0_disk2_6c4f554bdafa49baa780eb2d128ff39d"
+        },
+        {
+          "lun": 2,
+          "id": "/subscriptions/{subscription-id}/resourceGroups/myResourceGroup/providers/Microsoft.Compute/disks/vmss3176_vmss3176_2_disk3_7d5e664bdafa49baa780eb2d128ff38e"
+        }
+      ],
+      "detachDataDisks": [
+        {
+          "id": "/subscriptions/{subscription-id}/resourceGroups/myResourceGroup/providers/Microsoft.Compute/disks/vmss3176_vmss3176_1_disk1_1a4e784bdafa49baa780eb2d128ff65x",
+          "detachOption": "ForceDetach"
+        },
+        {
+          "id": "/subscriptions/{subscription-id}/resourceGroups/myResourceGroup/providers/Microsoft.Compute/disks/vmss3176_vmss3176_4_disk4_4d4e784bdafa49baa780eb2d256ff41z",
+          "detachOption": "ForceDetach"
+        }
+      ]
+    }
+  },
+  "responses": {
+    "202": {
+      "headers": {
+        "location": "https://foo.com/operationstatus"
```

Comment:
Is this correct example for location ? And also do we by default provide location in the response header ?

Reply:
The example used here is what is used generally in other example files.
For Long operations like Create/Update operations, we provide operation tracking link in location header.

Reply:
ok sure. If it is across same, will adhere to it, but still feel a better operationtracking url seems good.